valheimconfig.json

```diff
@@ -91,36 +91,36 @@
     "DisplayName":"Install BepInEx",
     "Category":"SteamCMD and Updates",
     "Description":"Installs BepInEx. Standalone - DO NOT use with the Install Valheim Plus option.",
-    "Keywords":"Install,'BepInEx'",
+    "Keywords":"Install,BepInEx",
     "FieldName":"bepinex_install",
     "InputType":"checkbox",
     "IsFlagArgument":false,
     "ParamFieldName":"bepinex_install",
     "IncludeInCommandLine":false,
     "DefaultValue":"false",
     "EnumValues":{
-        "False": "false",
-        "True": "true"
+        "False": "False",
+        "True": "True"
     }
   },{
     "DisplayName":"Install Valheim Plus",
     "Category":"SteamCMD and Updates",
-    "Description":"Installs Valheim Plus.  Includes BepInEx",
-    "Keywords":"Install,'Valheim Plus'",
+    "Description":"Installs Valheim Plus. Includes BepInEx",
+    "Keywords":"Install,Valheim,Plus",
     "FieldName":"valheim_plus_install",
     "InputType":"checkbox",
     "IsFlagArgument":false,
     "ParamFieldName":"valheim_plus",
     "IncludeInCommandLine":false,
     "DefaultValue":"false",
     "EnumValues":{
-        "False": "false",
-        "True": "true"
+        "False": "False",
+        "True": "True"

```

Comment:
The right side values can be lowercase still. It's just the left side that was what AMP expects to be "True" or "False". Changing this broke the update stages. :)